src/api/gallery.py

```diff
@@ -0,0 +1,127 @@
+"""
+Gallery Endpoints to Fetch, Upload and Remove image from a pro's gallery
+"""
+
+from flask import Blueprint, jsonify, request, abort
+import json
+import uuid
+import datetime
+from hashlib import md5
+import config
+from store import upload_blob, delete_blob
+from utils import load_json, save_json, create_thumbnail
+
+gallery = Blueprint('gallery', __name__)
+
+@gallery.route('/<string:pid>/gallery/', methods=['GET'])
+def fetch_gallery_details(pid):
+    """
+    Fetches gallery details of a professional
+    """
+    # load the database
+    try:
+        db = load_json(config.DB['JSON_PATH'])
+        if not db:
+            abort(404, description='No professionals found')
+    except (FileNotFoundError, PermissionError, json.JSONDecodeError) as e:
+        abort(500, description=str(e))
+
+    # check if the pro exists
+    if pid not in db:
+        abort(404, description='Professional %s not found' % pid)
+
+    return jsonify(db[pid]['gallery'])
+
+
+@gallery.route('/<string:pid>/gallery/', methods=['POST'])
+def upload_gallery(pid):
+    """
+    Uploads an image to the gallery container and its thumbnail to the thumbnail container of the pro
+    """
+    # load the database
+    try:
+        db = load_json(config.DB['JSON_PATH'])
+        if not db:
+            abort(404, description='No professionals found')
+    except (FileNotFoundError, PermissionError, json.JSONDecodeError) as e:
+        abort(500)
+
+    # check if the pro exists
+    if pid not in db:
+        abort(404, description='Professional %s not found' % pid)
+
+    # retrieve data from the request
+    image = request.files['image']
+    description = request.form['description'] if 'description' in request.form else None
```

Comment:
You can use `request.form.get('description')`. Also, too verbose names for low scope variables. Use `desc`.